lib/fluent/plugin/in_sql.rb

```diff
@@ -74,19 +74,27 @@ def init(tag_prefix, base_model, router, log)
         # creates a model for this table
         table_name = @table
         primary_key = @primary_key
+        time_formats = Fluent::VariableStore.fetch_or_build(:time_formats)
```

Comment:
I'm not comfortable using `Fluent::VariableStore`.
Because it's like a global variable, we should treat it carefully.
For example, the name `:time_formats` isn't appropriate, it might be used by other modules.
In Fluentd code, plugin name is used as name space for `Fluent::VariableStore`. e.g.) `:in_tail`
We should follow such naming convention.

On the other hand, I think we don't need to using `Fluent::VariableStore` here, setting a constant to the `@model` class is enough. I've added a commit for it: 9dee029f716d904ec14ae7797f17233b2ac76a80
Please check it @patrixr @cosmo0920 
I'll approve this pull request if it's no problem for you.

Reply:
Agreed @ashie, I much prefer this version 👍